virttest/test_setup.py

```diff
@@ -1465,6 +1466,43 @@ def is_binded_to_stub(self, full_id):
         stub_path = os.path.join(base_dir, "drivers/%s" % self.device_driver)
         return os.path.exists(os.path.join(stub_path, full_id))
 
+    def generate_ib_port_id(self):
+        """
+        Generate portid for the VF's of SR-IOV Infiniband Adapter
+        """
+        portid = "%s:%s" % (utils_net.generate_mac_address_simple(),
+                            utils_net.generate_mac_address_simple()[:5])
+        return portid
+
+    def set_linkvf_ib(self):
+        """
+        Set the link for the VF's in IB interface
+        """
+        pids = {}
+        try:
+            cmd = "ls -R /sys/class/infiniband/*/device/sriov/*"
+            dev = process.system_output(cmd, shell=True)
+        except process.CmdError as detail:
+            logging.error("No VF's found for set-up, test-failed as: %s",
+                          str(detail))
```

Comment:
1. %s/test-failed/command-failed
2. return `False` if it except.

Reply:
@balamuruhans made changes, please review, thanks!!!